src/Enum/AbortReason.ts

```diff
@@ -0,0 +1,10 @@
+export enum AbortReason {
+  WasCancelled, //0
+  AlreadyCalled, //1
+  BeforeCallWindow, //2
+  AfterCallWindow, //3
+  ReservedForClaimer, //4
+  InsufficientGas, //5
+  TooLowGasPrice,
```

Comment:
Did you mean to skip a comment on this line?

Reply:
I copy pasted this enum from Solidity EAC :)

Reply:
You pushed the comment for index 6 a line down.